### ISSUE TITLE
EOSTransport and leaving the lobby

### ISSUE THREAD

**Issue by martinpppx** [PlayEveryWare/eos_plugin_for_unity#190]
Hi, this is what my code looks like for a player to leave the lobby. 
```
private void Update() {
          if (Input.GetKeyDown(KeyCode.I)) {
                LobbyManager.LeaveLobby(LeaveCompleted);
          }
}

private void LeaveCompleted(Result result) {
            if (result == Result.Success) {
                NetworkManager.Singleton.Shutdown();
            }
}

```

But when I use it I get two errors in the console all the time, after shutting down the game I get a Unity crash.
![Screenshot_3](https://user-images.githubusercontent.com/49480579/210359543-49e473f6-28da-4f4c-98fa-ac3e40af2efb.png)
![Screenshot_4](https://user-images.githubusercontent.com/49480579/210359558-1147623f-e4e4-4078-b14b-5e04c8b15888.png)
![Screenshot_5](https://user-images.githubusercontent.com/49480579/210359576-b7a11a06-1f5c-4335-81c2-6fe9f9f51a4a.png)

Am I doing something wrong or is it just a bug?

**Comment by scovertb** [comment]
The lobbies and netcode transport aren't connected, so the LobbyManager use shouldn't be the issue. It looks like the error is the result of trying to close a connection with an invalid user, but from just that code snippet I'm not sure if it's a bug or an error on your part. The editor crash is almost definitely a bug. Could you reproduce the editor crash and then attach a copy of the editor crash log?

**Comment by scovertb** [comment]
That version shouldn't even have EOSTransport in it, so I'm not sure where that came from. It's possible you somehow ended up with a version of the EOS sample netcode that is missing some bug fixes. The next release should be the first one to officially include the netcode sample.

**Comment by martinpppx** [comment]
Well, that version of EOSTransport i got from that github: 
https://github.com/PlayEveryWare/eos_plugin_for_unity/tree/stable/Assets/Scripts/Networking

When will be released the next version of EOS plugin?

**Comment by scovertb** [comment]
The network transport sample is included in release 2.1.8 and should contain a fix for this issue.
